resources/lex-bot-aliases.go

```diff
@@ -0,0 +1,92 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/lexmodelbuildingservice"
+	"github.com/rebuy-de/aws-nuke/pkg/types"
+)
+
+type LexBotAlias struct {
+	svc      *lexmodelbuildingservice.LexModelBuildingService
+	name     *string
+	checksum *string
+	botName  *string
+}
+
+func init() {
+	register("LexBotAlias", ListLexBotAliases)
```

Comment:
Considering past naming conventions, other resources would suggest that the name be `LexModelBuildingServiceBotAlias` -- thoughts @svenwltr

Reply:
I agree